source/Color.cpp

```diff
@@ -130,3 +136,66 @@ Color Color::Multiply(float scalar, const Color &base)
 			scalar * base.color[2],
 			scalar * base.color[3]);
 }
+
+
+
+// Apply color blindness filters to this color.
+Color Color::Filter(Color c)
+{
+	Preferences::ColorFilter filter = Preferences::GetColorFilterMode();
+	// No color filters are enabled, so there's no need to adjust anything.
+	if(filter == Preferences::ColorFilter::NORMAL)
+		return c;
+
+	// Color blindness accessibility filters are enabled.
+	// LMS Daltonization is used to make the colors more distinct for
+	// color-blind players.
```

Comment:
Everywhere else here seems to be using "color blind" as two words. Any reason not to do the same here?
```suggestion
	// color blind players.
```